models/client_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestClientStatus(t *testing.T) {
+	testFiles := []string{
+		"status-single-basic.json",
+	}
+	for _, testFile := range testFiles {
+		status, err := GetStatusFromFile(testFile)
+		if err != nil {
+			t.Fail()
+		}
+		require.NoError(t, err)
```

Comment:
If you want to keep to the `assert` style I often use `assert.Nil(err)` you can then clean up the `if err != nil` and the `require.NoError`